snapshots/animal_welfare/2023-08-01/uk_egg_statistics.ods.dvc

```diff
@@ -0,0 +1,24 @@
+meta:
+  name: UK egg statistics
+  publication_year: 2023
+
+  publication_date: '2023-07-27'
+  source_name: Department for Environment, Food & Rural Affairs of the United Kingdom
+  source_published_by: Quarterly UK statistics on Egg Packing Station Throughput and
+    Prices, by the Department for Environment, Food & Rural Affairs of the United
+    Kingdom
+  url: https://www.gov.uk/government/statistics/egg-statistics
+  source_data_url: 
+    https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1173898/egg-packers-27july23.ods
+  license_url: 
+    https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1173898/egg-packers-27july23.ods
+  license_name: Public domain
+  date_accessed: 2023-08-01
+  is_public: true
+  description: |
+    DEFRA runs a quarterly survey of registered UK egg packing stations. It is a voluntary sample survey of 27 respondents that collects information on throughput by production type and prices of graded eggs and sales of ungraded eggs. The response rate is typically 100 per cent and the survey accounts for 75 per cent of eggs packed in the UK. The survey figures are raised up to give UK estimates using information on the number of commercial laying hens, average egg yields, average mortality rates, the proportion of UK eggs that go through packing stations. Throughput by egg type for packing stations not surveyed is calculated using data provided by packing stations responding to the survey. The raised figures are published in this statistics notice and the associated datasets. The figures in this notice therefore represent all Class A eggs passed over a grader in the UK, including seconds.
```

Comment:
Maybe not important but I'm not sure all of this is relevant and it's not totally clear to me what the 'seconds' in the last sentence refers to.

Reply:
I don't understand this explanation either. I think "seconds" means "[Class B](https://www.farminguk.com/content/knowledge/Grading%20your%20eggs.pdf)". But the sentence doesn't make sense to me.